### ISSUE TITLE
About the performance

### ISSUE THREAD

**Issue by Mrkkew** [Chenfeng1271/MGMRA#1]
I have run your code, but I don't get the performance of your paper. How to set the hyper-parameter to get the best performance?

**Comment by Chenfeng1271** [comment]
Please tell me the hyper-parameters you used. I think these results on RegDB are easy to achieve, while on SYSU-MM01, this result may be unstable. This phenomenon comes from its original code, sometimes the initialization would result in it.

**Comment by Mrkkew** [comment]
For RegDB，I set the --batch-size to 8 and --num_pos to 4, and I run train_MGMRA.py. I got Rank-1: 93.11% / mAP: 83.70%. 
For SYSU-MM01，I set the --batch-size to 8 and --num_pos to 6, and I run train_MGMRA.py. I got Rank-1: 60.58% / mAP: 56.15%. 
The learning rate is initialized to 0.1.
 The results is different from your paper. So what could be the problem ?

**Comment by Chenfeng1271** [comment]
The parameter seems to be identical to mine, but I got the results in my paper. I will upload the weight file after CVPR deadline, please wait.

**Comment by Chenfeng1271** [comment]
> For RegDB，I set the --batch-size to 8 and --num_pos to 4, and I run train_MGMRA.py. I got Rank-1: 93.11% / mAP: 83.70%. For SYSU-MM01，I set the --batch-size to 8 and --num_pos to 6, and I run train_MGMRA.py. I got Rank-1: 60.58% / mAP: 56.15%. The learning rate is initialized to 0.1. The results is different from your paper. So what could be the problem ?

Today I rerun my code on RegDB with visiable2infarared setting, it is easy to achieve 94% performance.

**Comment by Chenfeng1271** [comment]
![image](https://user-images.githubusercontent.com/24824628/141117264-4c1d2eb9-008c-49bb-ad0b-9fc3484289d0.png)
![image](https://user-images.githubusercontent.com/24824628/141117447-a6407867-d15f-4dd2-ac77-b78857af6aa1.png)
![image](https://user-images.githubusercontent.com/24824628/141117513-1a70ed18-26ba-4a57-bf0f-6ed3db4a7bf4.png)
![image](https://user-images.githubusercontent.com/24824628/141117585-92a958a3-7803-4188-a3c1-c0a3d987e1df.png)
![image](https://user-images.githubusercontent.com/24824628/141117670-26411af9-033b-471f-9cb8-8967293c4b3b.png)
![image](https://user-images.githubusercontent.com/24824628/141117737-e73761ab-42fb-4339-99fb-497272cf2a17.png)
These are performance screencut from trial 1-6

**Comment by Chenfeng1271** [comment]
![image](https://user-images.githubusercontent.com/24824628/141296900-2a649895-e14f-4289-a4d3-96541c002aa2.png)
![image](https://user-images.githubusercontent.com/24824628/141296977-8eb4f8f9-a651-41c2-90d8-eeefd3185633.png)
I train my model on SYSU twice, and these are performance screencut

**Comment by ssyou** [comment]
Hi, I have evaluated the SYSU-MM01 dataset with the provided model but got Rank-1: 58.72% / mAP: 54.59%. 
Furthermore, the screencut provided by the author is the validation result, not the test result.
![image](https://user-images.githubusercontent.com/43953749/162209033-53a1701f-0a63-4db2-a967-fec282877598.png)

**Comment by Mrkkew** [comment]
I think you should carefully check the experiment evaluations in cross-modality person ReID. The results shown in the response are the validation result, not the test result. The results on the SYSU-MM01 dataset are an average performance of 10 times repeated random probe/gallery splits, while that on the RegDB dataset are an average performance of 10 trials with different splits of training/testing sets.

**Comment by Mrkkew** [comment]
@Chenfeng1271
